### PR TITLE
Feature/2097 outreach report

### LINKED ISSUE #2097: Outreach report
We currently ask candidates in the application where they heard about the role to aid future outreach. This information is pulled across into an outreach report for each exercise. Additionally, as part of the diversity monitoring form we ask candidates whether they have attended an outreach event, participated in the Pre-application Judicial Education Programme or engaged in judicial shadowing. It would be beneficial to have all of this information in one place and to, therefore, add the additional fields from the diversity monitoring form to the outreach report on the platform.

As of 21st Sep:
In addition, the declaration rate on the report is currently listed amongst the outreach options. At the top of the report is a grey box showing the 'Total applications'. Could the grey box also include the declaration rate?

### PR DESCRIPTION
**Pull request by drieJAC**
## What's included?

**Only Sean Cuddihy needs to test this!**

https://jac-apply-admin-production--2097-outreach-j3l1v5bc.web.app [expires 2023-10-20 11:09:48]

**Please remember to press the 'refresh' button for the report!**

We currently ask candidates the following questions when applying:

-  where they heard about the role
- have they attended an outreach event on JAC selection exercises
- did they participate in a Judicial Workshadowing Scheme
- have they taken part in the Pre-Application Judicial Education Programme (PAJE)

The outreach report collates the figures for the answers to these questions.

Closes #2097 

## Who should test?
Sean Cuddihy

## How to test?
Go to the Outreach Report and ensure the report numbers accurately reflect the answers from the applications.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Comment by nickaddy** [comment]
It would be beneficial to have all of this information in one place and to, therefore, add the additional fields from the diversity monitoring form to the outreach report on the platform.

In addition, the declaration rate on the report is currently listed amongst the outreach options. At the top of the report is a grey box showing the 'Total applications'. Could the grey box also include the declaration rate?

File: src/views/Exercise/Reports/Outreach.vue [hunk 1]
```diff
@@ -123,23 +119,254 @@
...
+              <td class="govuk-table__cell govuk-table__cell--numeric">
+                <Stat :stat="report[activeTab].outreach['judicial-office-extranet']" />
+              </td>
+            </tr>
+            <tr class="govuk-table__row">
+              <th class="govuk-table__header">
+                Judging Your Future Newsletter
+              </th>
+              <td class="govuk-table__cell govuk-table__cell--numeric">
+                <Stat :stat="report[activeTab].outreach['judging-your-future-newsletter']" />
```
Comment (warrensearle) [inline src/views/Exercise/Reports/Outreach.vue:162]: Do we _need_ to hardcode this? Looks like it was dynamic previously and now has been changed to be hardcoded. Please double check

**Review by warrensearle — CHANGES_REQUESTED** [review]

File: src/views/Exercise/Reports/Outreach.vue (hunk 1 above)
Comment (drieJAC) [reply src/views/Exercise/Reports/Outreach.vue]: Done

**Comment by drieJAC** [comment]
Feedback from Sean Cuddihy:

On the production link in your email (https://jac-apply-admin-production--2097-outreach-j3l1v5bc.web.app/) some exercises seemed to be working (00129 worked after I pressed refresh), but 00162 gave an error after I pressed refresh (see screenshot) and I tried an older archived exercise 00115 and that didn't have a refresh button at all.

**Comment by drieJAC** [comment]
So there was a memory issue with 00162. We have increased the memory and I was able to hit the refresh button fine. That exercise has under 2000 applications and we are ideally supporting 5000 so we are going to do some work to make the memory usage more efficient
Regarding the missing Refresh button for 00115 I didn't see this. I was able to press the button and the numbers got generated ok
I'll make changes to the PAJE question so that the 'online only' and 'judge-led' are displayed and i'll give you a shout as soon as that's done

**Review by tomlovesgithub — APPROVED** [review]

**Review by HalcyonJAC — APPROVED** [review]

**Pull request merged**